File: app/src/main/java/com/fitnessapp/View/Fragments/ExerciseFragment.java
package com.fitnessapp.View.Fragments;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.fitnessapp.Controll.MainController;
import com.fitnessapp.View.MainActivity;

import ownuse.fitnessapp.R;

public class ExerciseFragment extends Fragment{

    private MainController mainController;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_newexercise, container, false);

        mainController = MainActivity.mainController;


        return view;
    }

    //TODO: everything
}

File: app/src/main/java/com/fitnessapp/View/Fragments/ExerciseFragment.kt
package com.fitnessapp.View.Fragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.fitnessapp.Controll.MainController

import ownuse.fitnessapp.R

class ExerciseFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater!!.inflate(R.layout.fragment_newexercise, container, false)


        return view
    }

    //TODO: everything
}

